tour_model_eval/v-measurel_cutoff_bins_single_user.ipynb

```diff
@@ -138,47 +141,58 @@
     "    print(bin_df)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "expensive-forestry",
+   "metadata": {},
+   "source": [
+    "### Original output"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "tired-athens",
+   "id": "corporate-situation",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# turn all user_input into list without binning\n",
-    "bin_trips_user_input_ls = pd.DataFrame(data=[bin_trips[i][\"data\"][\"user_input\"] for i in range(len(bin_trips))]).values.tolist()\n",
-    "bin_trips_user_input_ls"
+    "bin_trips_df = pd.DataFrame(data=[i[\"data\"][\"user_input\"] for i in bin_trips])\n",
+    "bin_trips_df"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "veterinary-courage",
+   "id": "historic-russian",
    "metadata": {},
    "outputs": [],
    "source": [
-    "for i in bin_trips_user_input_ls:\n",
-    "    if i[2] == \"same_mode\":\n",
-    "        print(i)"
+    "# turn all user_input into list without binning\n",
+    "bin_trips_user_input_ls = bin_trips_df.values.tolist()\n",
+    "bin_trips_user_input_ls"
```

Comment:
where are you using this? can you just delete it?

Reply:
I use `bin_trips_user_input_ls` to get `labels_true` because I don't know how to directly check if one row in a data frame is also in another data frame and get the index from another data frame.